.github/workflows/unit-tests.yml

```diff
@@ -199,8 +199,9 @@ jobs:
           BIONEMO_DATA_SOURCE: ngc
           # this variable should be used in the notebooks to run a subset of the model layers or a smaller model/dataset
           FAST_CI_MODE: true
+        # FIXME:  remove --ignore when this issue is fixed: https://github.com/NVIDIA/bionemo-framework/issues/778
         run: |
-          pytest -v --nbval-lax -p no:python docs/ sub-packages/
+          pytest -v --nbval-lax -p no:python --ignore="docs/docs/user-guide/examples/bionemo-geneformer/geneformer_cellxgene_tutorial.ipynb" docs/ sub-packages/
```

Comment:
This might get cumbersome eventually, I wonder if we could put this list in a file somewhere and read it in? Or a conftest.py

Reply:
i was worried that conftest with allow us to collect more of those items but I will test if it works since it is more elegant solution